README.rst

```diff
@@ -3,12 +3,33 @@
 QSIprep: Preprocessing and analysis of q-space images
 =======================================================
 
+.. image:: https://img.shields.io/badge/Source%20Code-pennlinc%2Fqsiprep-purple
+  :target: https://github.com/PennLINC/qsiprep
+  :alt: GitHub Repository
+
 .. image:: https://readthedocs.org/projects/qsiprep/badge/?version=latest
   :target: http://qsiprep.readthedocs.io/en/latest/?badge=latest
   :alt: Documentation Status
 
+.. image:: https://img.shields.io/badge/docker-pennbbl/qsiprep-brightgreen.svg?logo=docker&style=flat
+  :target: https://hub.docker.com/r/pennbbl/qsiprep/tags/
+  :alt: Docker
+
 .. image:: https://circleci.com/gh/PennLINC/qsiprep/tree/master.svg?style=svg
   :target: https://circleci.com/gh/PennLINC/qsiprep/tree/master
+  :alt: Test Status
+
+.. image:: https://codecov.io/gh/PennLINC/qsiprep/branch/master/graph/badge.svg
```

Comment:
This just says "unknown", maybe we add it after we get codecov up and running?

Reply:
Sure. I've removed it for now.